src/doc/using_visit/Quantitative/Query.rst

```diff
@@ -445,18 +472,18 @@ XRay Image
 
     When specifying "bov" output, 2 files are created for each variable.
     One contains the ``intensity`` and the other the ``path_length``.
-    The files are named ``outputXX.bof`` and ``outputXX.bov`` with ``XX`` being a sequence number.
+    The files are named ``output.XX.bof`` and ``output.XX.bov`` with ``XX`` being a sequence number.
```

Comment:
here you are using two `XX` chars which doesn't seem to jive with other examples above. Why?

Reply:
The behavior of output filenames is very idiosyncratic. Those `XX`s do not refer to cycle info or familying. Rather, some outputs from the query come out as multiple files. These files, if familying is enabled, will have the same family number, (`XXXX`), but they also get another number to differentiate them. Similarly, they will all get the same cycle number, but they have this extra extension as well.

Reply:
Ah, ok. Jus curious but is the number of these always known to be less than 100?

Reply:
I don't have reason to believe it is. That being said, I have not seen a use of the x ray query that has that many. Even Steve's use cases that generate many bov files did not go past that limit.

This is an area that could (and probably should) be improved. We are in danger of falling off the end here and with the file familying implementation. If you have `output.0000.png` all the way through `output.9999.png`, the x ray query will start overwriting `output.9999.png` repeatedly, every time it is called.

Reply:
To close the loop on this, I've added both to my list of continued enhancements for the x ray query.